packages/codemirror/src/Editor.ts

```diff
@@ -16,7 +16,9 @@ export class Editor extends HTMLWidget {
     options(): any {
         return {
             lineNumbers: true,
-            tabSize: 2
+            tabSize: 2,
+            gutters: ["CodeMirror-lint-markers"],
+            lint: true
```

Comment:
I think the show "lint" should be optional "showHighlightGutter"?

Reply:
I don't recall why I added "lint" in this pull request. Did we discuss it? It seems like it's beyond the scope of this change and would need this addon to be loaded (in codemirror-shim?):
https://codemirror.net/addon/lint/lint.js

As well as these addons (as needed?):
https://codemirror.net/addon/lint/javascript-lint.js
https://codemirror.net/addon/lint/json-lint.js
https://codemirror.net/addon/lint/css-lint.js
https://codemirror.net/addon/lint/html-lint.js